config/brokers.toml

```diff
@@ -14,14 +14,26 @@ secret = ""
 [ib]
 host = "127.0.0.1"
 
+# when clients are being scanned this determines
+# which clients are preferred to be used for data
+# feeds based on the order of account names, if
+# detected as active on an API client.
+prefer_data_account = [
+    'paper',
+    'margin',
+    'ira',
+]
+
+# the order in which ports will be scanned
+# (by the `brokerd` daemon-actor)
+# is determined # by the line order here.
 ports.gw = 4002
```

Comment:
Got a feeling this is still broken when scanning for more then one port # (can't member why tho) so probably needs at least some manual testing before merge..

Reply:
Well testing says otherwise once I tweaked a couple things 😂 

At least one more commit coming, i might also toss in a commit that changes the feed status label thing on the sidepane to show the current "data client" socket info.